build/cli.ts

```diff
@@ -6,21 +6,36 @@ import zlib from "node:zlib";
 
 import chalk from "chalk";
 import cliProgress from "cli-progress";
-import { program } from "@caporal/core";
-import { prompt } from "inquirer";
-
-import { Document, slugToFolder, translationsOf } from "../content";
-import { CONTENT_ROOT, CONTENT_TRANSLATED_ROOT } from "../libs/env";
-import { VALID_LOCALES } from "../libs/constants";
-import { renderHTML } from "../ssr/dist/main";
-import options from "./build-options";
-import { buildDocument, BuiltDocument, renderContributorsTxt } from ".";
-import { Flaws } from "../libs/types";
+import caporal from "@caporal/core";
+import inquirer from "inquirer";
+
+import {
+  Document,
+  slugToFolder,
+  translationsOf,
+  MEMOIZE_INVALIDATE,
+} from "../content/index.js";
+import {
+  CONTENT_ROOT,
+  CONTENT_TRANSLATED_ROOT,
+  BUILD_OUT_ROOT,
+} from "../libs/env/index.js";
+import { VALID_LOCALES } from "../libs/constants/index.js";
+import { renderHTML } from "../ssr/dist/main.js";
+import options from "./build-options.js";
+import {
+  buildDocument,
+  BuiltDocument,
+  renderContributorsTxt,
+} from "./index.js";
+import { Flaws } from "../libs/types/document.js";
 import * as bcd from "@mdn/browser-compat-data/types";
-import SearchIndex from "./search-index";
-import { BUILD_OUT_ROOT } from "../libs/env";
-import { makeSitemapXML, makeSitemapIndexXML } from "./sitemaps";
-import { humanFileSize } from "./utils";
+import SearchIndex from "./search-index.js";
+import { makeSitemapXML, makeSitemapIndexXML } from "./sitemaps.js";
+import { humanFileSize } from "./utils.js";
+
+const { program } = caporal;
```

Comment:
Why was this change necessary? In doubt `import { default as program } from "@caporal/core"` should work.

Reply:
`program` isn't the default export from `@caporal/core`; otherwise we could just do `import program from "@caporal/core"`.  The reason this is needed is because Caporal is a CommonJS package, and as such `program` isn't a separate export, but rather a property on the default export.

(Side note: Caporal hasn't gotten an update in two years.  We may want to look into a replacement anyways.)

Reply:
Oh, that makes sense, thanks for the clarification.

> (Side note: Caporal hasn't gotten an update in two years. We may want to look into a replacement anyways.)

Good point, I guess we'll want to migrate to [commander.js](https://www.npmjs.com/package/commander).

Reply:
Thank you for opening https://github.com/mdn/yari/issues/7800 for this. 🎉